src/omni_epd/displays/inky_display.py

```diff
@@ -49,6 +48,10 @@ def __init__(self, deviceName, config):
         elif(dType == 'what'):
             deviceObj = self.load_display_driver(self.pkg_name, 'what')
             self._device = deviceObj.InkyWHAT(dColor)
+        elif(dType == 'auto'):
+            deviceObj = self.load_display_driver(self.pkg_name, 'auto')
+            self._device = deviceObj.auto()
+            dColor = 'color' if self._device.colour == 'multi' else self._device.colour

```

Comment:
```suggestion
            if self._device.colour == 'multi':
                from inky.inky_uc8159 import DESATURATED_PALETTE
                self.mode = dColor = InkyImpressionDisplay.mode
                self.modes_available = InkyImpressionDisplay.modes_available
                self.max_colors = InkyImpressionDisplay.max_colors
                self.palette_filter = DESATURATED_PALETTE
                InkyDisplay._display = InkyImpressionDisplay._display
                InkyDisplay.clear = InkyImpressionDisplay.clear
            else:
                dColor = self._device.colour
```
Got inky impression working

Reply:
Comparing this to the files in this PR now we are missing the setting of the max_colors, palette, etc. I don't have one of these - how important are setting those values? I'm trying to envision end-user issues where they try and access `epd.max_colors` and it's not correct. Ultimately this is a stepping stone to other projects so the main object attributes should be correct even if we're not using them directly here right? 

The Impression also uses a different method of Clearing the display (clean vs white). I notice in the below code you've added checks for that when merging the classes together.

Reply:
After thinking about it some more, `max_colors` is only used to used to [limit the number of colors](https://github.com/robweber/omni-epd/blob/25a365e79668b13b8ad45d8ea6350ae2ed9f8190/src/omni_epd/virtualepd.py#L190-L192) allowed in palette_filter. And inky impression is limited to black, white, green, blue, red, yellow, and orange. An image's palette will always display as those respective colors, regardless of what they actually are, so I'm not sure how useful the ability to set custom palettes would be.

We're not filtering the image in color mode, so `palette_filter` isn't even used in that case.